libsks/src/pkcs11_token.c

```diff
@@ -66,6 +66,7 @@ CK_RV sks_ck_slot_get_list(CK_BBOOL present,
 
 	if (!slots || *count < (size / sizeof(uint32_t))) {
 		*count = size / sizeof(uint32_t);
+		if(!slots) return CKR_OK;
```

Comment:
Nice, thanks.
Can you expand over several line, for consistency of the code layout:
```
if (!slot || *count < (size / sizeof(uint32_t))) {
    *count = size / sizeof(uint32_t);
    if (!slots)
        return CKR_OK;
   return CKR_BUFFER_TOO_SMALL;
}
```